examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/models/IBNMeta.py

```diff
@@ -0,0 +1,222 @@
+from __future__ import absolute_import
```

Comment:
If these code is a copy of opensource repository, please keep the original copyright here.